packages/manager/modules/pci/src/projects/project/storages/object-storage/users/import/import.controller.js

```diff
@@ -0,0 +1,68 @@
+export default class PciBlockStorageContainersContainerObjectAddController {
+  /* @ngInject */
+  constructor($http, $translate, PciStoragesObjectStorageService) {
+    this.$http = $http;
+    this.$translate = $translate;
+    this.PciStoragesObjectStorageService = PciStoragesObjectStorageService;
+  }
+
+  $onInit() {
+    this.isLoading = false;
+    this.files = [];
+  }
+
+  static isJson(str) {
```

Comment:
May be we can use `isJSON` lodash function.

Reply:
I dont see isJSON from lodash @https://lodash.com/docs/4.17.15